client/src/quic_client.rs

```diff
@@ -65,21 +65,21 @@ impl TpuConnection for QuicTpuConnection {
         &self.client.addr
     }
 
-    fn send_wire_transaction(&self, data: Vec<u8>) -> TransportResult<()> {
+    fn send_wire_transaction(&self, data: &[u8]) -> TransportResult<()> {
         let _guard = self.client.runtime.enter();
-        let send_buffer = self.client.send_buffer(&data[..]);
+        let send_buffer = self.client.send_buffer(data);
         self.client.runtime.block_on(send_buffer)?;
         Ok(())
     }
 
-    fn send_batch(&self, transactions: Vec<Transaction>) -> TransportResult<()> {
+    fn send_batch(&self, transactions: &[Transaction]) -> TransportResult<()> {
         let buffers = transactions
             .into_par_iter()
             .map(|tx| bincode::serialize(&tx).expect("serialize Transaction in send_batch"))
             .collect::<Vec<_>>();
 
         let _guard = self.client.runtime.enter();
-        let send_batch = self.client.send_batch(&buffers[..]);
+        let send_batch = self.client.send_batch(&buffers);
```

Comment:
Since we're using &buffers instead of &buffers[..] can we change QuicClient::send_batch and all other functions in QuicClient to do the same?

Reply:
Yeah. send_batch is the only function in that file having this nits in quic_client. udp_client has another. will fix